lib/api/middleware/set-linz-model.js

```diff
@@ -2,14 +2,42 @@
 
 const linz = require('../../../');
 
+/**
+ * Get the model name from a request.
+ * @param {Object} req The HTTP request object.
+ * @returns {String} The model name.
+ */
+const getModelFromRequest = (req) => {
```

Comment:
This isn't middleware. Please move it to `linz.api.model` and call it `fromRequest`.

Reply:
@smebberson This expands upon `const model = req.params.model || req.params.config;` and is only intended to be used in this file. It is being exported so I could write some tests for the function.

Reply:
Hmm. I can see what you're saying. You generally don't test stuff that isn't exposed though (good testing practices), so the fact that you're exposing it to test either means, we shouldn't have those tests, or we should expose it (and in this case move it to a proper location).

I think it's a good bit of function, why not have it expose in `linz.api.model`?